build/templates/_converters.py.mako

```diff
@@ -8,6 +8,32 @@ import datetime
 import six
 
 
+def _string_to_list(string, prefix):
+    '''Convert a string to a list of repeated capabilities numbers I.e. no prefix
+
```

Comment:
I think you should add a big fat note about this function duplicating logic in our drivers, and having to keep up with changes and deal with versioning issues should any of that change or be inconsistent across MI drivers.